src/bp-core/admin/bp-core-admin-settings.php

```diff
@@ -853,7 +854,39 @@ function bp_feed_settings_callback_post_type( $args ) {
 		<p class="description"><?php _e( 'When users publish new blog posts, show them in the activity feed.', 'buddyboss' ); ?></p>
 		<?php
 	}
+}
+
+/**
+ * Allow activity comments on posts and comments.
+ *
+ * @since BuddyBoss 1.5.9
+ */
+function bp_feed_settings_callback_post_type_comments( $args ) {
+	
+	$post_type     = $args['post_type'];
+	$option_name   = bp_post_type_feed_comment_option_name( $post_type );
+	$post_type_obj = get_post_type_object( $post_type );
+	?>
 
+	<input
+		class="bp-feed-post-type-commenet-checkbox <?php echo 'bp-feed-post-type-comment-' . $post_type; ?>"
+		name="<?php echo $option_name; ?>"
+		id="<?php echo $option_name; ?>"
+		type="checkbox"
+		value="1"
+		<?php checked( bp_is_post_type_feed_comment_enable( $post_type, false ) ); ?>
+	/>
+	<label for="<?php echo $option_name; ?>">
+		<?php echo $post_type === 'post' ? esc_html__( 'Show blog post comments', 'buddyboss' ) : 'Enable comments in ' . $post_type_obj->labels->name . ' activity posts.'; ?>
```

Comment:
`'Enable comments in '` need translation functions

Reply:
Set text domain https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/7f966a46e05f70f49eb0c5432da644b3ecce6fee